common/models/acl.js

```diff
@@ -489,9 +612,9 @@ module.exports = function(ACL) {
       modelId: modelId,
     });
 
-    this.checkAccessForContext(context, function(err, access) {
+    this.checkAccessForContext(context, function(err, accessRequest) {
       if (err) callback(err);
-      else callback(null, access.permission !== ACL.DENY);
+      else callback(null, accessRequest.isAllowed());
     });
```

Comment:
the existing `isAllowed` instance method of AccessRequest should be used here for more clarity instead of checking value of nested permission